openquake/hazardlib/gsim/dost_2004.py

```diff
@@ -0,0 +1,157 @@
+# The Hazard Library
+# Copyright (C) 2015 GEM Foundation
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+Module exports :class:'DostEtAl2004'
+"""
+from __future__ import division
+import numpy as np
+# standard acceleration of gravity in m/s**2
+from scipy.constants import g
+
+
+from openquake.hazardlib.gsim.base import GMPE, CoeffsTable
+from openquake.hazardlib import const
+from openquake.hazardlib.imt import PGA, PGV
+
+class DostEtAl2004(GMPE):
+    """
+    Implements the GMPE of Dost et al. (2004) for PGA and PGV from
+    induced seismicity earthquakes in the Netherlands
+    Dost, B., van Eck, T. and Haak, H. (2004) Scaling of peak ground
+    acceleration and peak ground velocity recorded in the Netherlands.
+    Bollettino di Geofisica Teorica ed Applicata. 45(3), 153 - 168
+    """
+    #: The GMPE is derived from induced earthquakes
+    DEFINED_FOR_TECTONIC_REGION_TYPE = const.TRT.INDUCED
+
+    #: Supported intensity measure types are peak ground acceleration
+    #: and peak ground velocity
+    DEFINED_FOR_INTENSITY_MEASURE_TYPES = set([
+        PGA,
+        PGV
+    ])
+
+    #: Supported intensity measure component is the average horizontal
+    DEFINED_FOR_INTENSITY_MEASURE_COMPONENT = const.IMC.AVERAGE_HORIZONTAL
```

Comment:
The component in Dost et al. (2004) is average horizontal, however in the Dost adaptation is geometric mean of the rotated horizontals.